### ISSUE TITLE
free accounts

### ISSUE THREAD

**Issue by j-mizzle** [xou816/spot#491]
are there plans to allow free accounts to be able to sign in and use Spot? I keep a music collection at home so I rarely stream, which makes signing up and paying monthly for premium hard to justify. I currently use Pithos (Pandora client) on my Pinephone but it's rather limited. Is there a particular reason to only allow premium accounts? Blade Player on Android allows free Spotify accounts to sign in also, so I'm curious about this limitation.

**Comment by Toorero** [comment]
> Note: [librespot](https://github.com/librespot-org/librespot) only works with Spotify Premium. This will remain the case. We will not support any features to make librespot compatible with free accounts, such as limited skips and adverts.
That's why

**Comment by j-mizzle** [comment]
ahh, sorry I didn't see that note. I've actually never used the official Spotify client as before I switched to using my Pinephone I had been running Android without Google apps for years. I guess I never noticed the limited skips while using Blade Player to access Spotify since all the songs on my playlists was added by me and liked so I never really skipped anything. I have however never once heard an ad using Blade, but I suppose that could be due to some hackery or API they use.
Thanks for the quick answer!
